package.json

```diff
@@ -9,21 +9,30 @@
     "typescript": "4.7.4"
   },
   "scripts": {
-    "precommit": "npm run lint & lerna run test",
     "lint": "eslint packages/**/*.ts",
+    "test": "lerna run test",
+    "test:swr": "lerna run test --scope @datx/swr",
+    "bootstrap": "lerna bootstrap",
```

Comment:
Lerna bootstrap does not work with yarn 2+ and is unnecessary. We have even remove that step from CONTRIBUTING.md in [this PR](https://github.com/infinum/datx/pull/1101/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055L22) so you can remove this script as well

Related issue: https://github.com/infinum/datx/issues/1098